modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/Testcontainers.java

```diff
@@ -0,0 +1,50 @@
+package org.testcontainers.junit.jupiter;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * {@code @Testcontainers} is a JUnit Jupiter extension to activate automatic
+ * startup and stop of containers used in a test case.
+ *
+ * <p>The test containers extension finds all fields of type
+ * {@link org.testcontainers.lifecycle.Startable} and calls their container
+ * lifecylce methods. Containers declared as static fields will be shared
+ * between test methods. They will be started only once before any test method
+ * is executed and stopped after the last test method has executed. Containers
+ * declared as instance fields will be started and stopped for every test
+ * method.</p>
+ *
+ * <p>Example:
+ *
+ * <pre>
```

Comment:
I think `<pre>` tag inside `<p>` tag is breaking javadoc generation, see:
https://travis-ci.org/testcontainers/testcontainers-java/jobs/438706848
```
> Task :junit-jupiter:javadoc FAILED
/home/travis/build/testcontainers/testcontainers-java/modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/Testcontainers.java:44: error: unexpected end tag: </p>
 * </p>
   ^
1 error
FAILURE: Build failed with an exception.
```